### ISSUE TITLE
Urwolf doesn't count as Werwolf

### ISSUE THREAD

**Issue by Toaster2-0** [foin137/werwolfonline.eu#12]

**Comment by Toaster2-0** [comment]
(If you look at him as Spion)

**Comment by foin137** [comment]
Right, but I'm not sure if this is actually a bug or a feature ;-)

What I mean is:
Should the "Urwolf" be recognized as a "Werwolf" as he is actually a different character/identity?

I mean both makes sense:
One might argue that "Urwolf" should be recognized as a "Werwolf" because he is in their team and acts as a "Werwolf",
on the other hand, the "Hexe" for example is also not recognized as "Dorfbewohner" because it is technically a different character.

**Comment by Joshix-1** [comment]
I would say for a "Spion" it shouldn't be a "Werwolf", but for the "Paranormaler Entwickler" it should be. The Spion checks for the character not the team.

**Comment by foin137** [comment]
That makes sense - should change that
